content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -171,6 +171,13 @@ There are two possible values:
 - `onstart`: The APIService should be reconciled when an API server starts up, but not otherwise.
 - `true`: The API server should reconcile this APIService continuously.
 
+### service.alpha.kubernetes.io/tolerate-unready-endpoints (deprecated)
+
+Used on: StatefulSet
+
+This annotation on the Service is denoting if the endpoints controller should go ahead and create endpoints for unready pods.
+Endpoints of these Services retain their DNS records and continue receiving traffic for the Service from the moment the kubelet starts all containers in the pod and marks it "Running", till the kubelet stops all containers and deletes the pod from the apiserver.
```

Comment:
Please wrap the long lines as we did for other items.

Reply:
@tengqm i have Addressed the changes . PTAL!!